### PR TITLE
Add OpenAPI documentation for models

### PR DESCRIPTION
**Pull request by reugn**

### CODE REVIEW

File: pkg/model/aerospike_cluster.go [hunk 1]
```diff
@@ -8,16 +8,24 @@ import (
...
-	AuthMode             *string `yaml:"auth-mode,omitempty" json:"auth-mode,omitempty"`
+	pwdOnce sync.Once
+	pwd     *string
+	// The host that acts as the entry point to the cluster.
+	Host *string `yaml:"host,omitempty" json:"host,omitempty"`
+	// The port to connect to.
+	Port *int32 `yaml:"port,omitempty" json:"port,omitempty"`
+	// Whether should use "services-alternate" instead of "services" in info request during cluster tending.
+	UseServicesAlternate *bool `yaml:"use-services-alternate,omitempty" json:"use-services-alternate,omitempty"`
+	// The usename for the cluster authentication.
```
Comment (korotkov-aerospike) [inline pkg/model/aerospike_cluster.go:21]:
```suggestion
	// The username for the cluster authentication.
```

File: pkg/model/backup_policy.go [hunk 2]
```diff
@@ -3,23 +3,42 @@ package model
...
+	// Maximum number of scan calls to run in parallel.
+	Parallel *int32 `yaml:"parallel,omitempty" json:"parallel,omitempty"`
+	// Socket timeout in milliseconds. If this value is 0, it is set to total-timeout.
+	// If both are 0, there is no socket idle time limit.
+	SocketTimeout *uint32 `yaml:"socket-timeout,omitempty" json:"socket-timeout,omitempty"`
+	// Total socket timeout in milliseconds. Default is 0, that is, no timeout.
+	TotalTimeout *uint32 `yaml:"total-timeout,omitempty" json:"total-timeout,omitempty"`
+	// Maximum number of retries before aborting the current transaction.
+	MaxRetries *uint32 `yaml:"max-retries,omitempty" json:"max-retries,omitempty"`
+	RetryDelay *uint32 `yaml:"retry-delay,omitempty" json:"retry-delay,omitempty"`
```
Comment (korotkov-aerospike) [inline pkg/model/backup_policy.go:17]:
```suggestion
        // RetryDelay defines the delay in milliseconds before retrying a failed operation.
	RetryDelay *uint32 `yaml:"retry-delay,omitempty" json:"retry-delay,omitempty"`
```

File: pkg/model/backup_policy.go [hunk 3]
```diff
@@ -3,23 +3,42 @@ package model
...
+	MaxRecords *uint64 `yaml:"max-records,omitempty" json:"max-records,omitempty"`
+	// Limit total returned records per second (RPS). If RPS is zero (the default),
+	// the records-per-second limit is not applied.
 	RecordsPerSecond *uint32 `yaml:"records-per-second,omitempty" json:"records-per-second,omitempty"`
-	FileLimit        *uint64 `yaml:"file-limit,omitempty" json:"file-limit,omitempty"`
-	FilterExp        *string `yaml:"filter-exp,omitempty" json:"filter-exp,omitempty"`
+	// File size limit (in MB) for --directory. If an .asb backup file crosses this size threshold,
+	// a new backup file will be created.
+	FileLimit *uint64 `yaml:"file-limit,omitempty" json:"file-limit,omitempty"`
+	FilterExp *string `yaml:"filter-exp,omitempty" json:"filter-exp,omitempty"`
```
Comment (korotkov-aerospike) [inline pkg/model/backup_policy.go:41]:
```suggestion
        // custom b64-encoded filter expression to use in the scan calls
	FilterExp *string `yaml:"filter-exp,omitempty" json:"filter-exp,omitempty"`
```

File: pkg/model/backup_routine.go [hunk 4]
```diff
@@ -5,24 +5,35 @@ import (
...
+	SourceCluster string `yaml:"source-cluster,omitempty" json:"source-cluster,omitempty"`
+	// The name of the corresponding storage provider configuration.
+	Storage string `yaml:"storage,omitempty" json:"storage,omitempty"`
+	// The Secret Agent configuration for the routine (optional).
+	SecretAgent *string `yaml:"secret-agent,omitempty" json:"secret-agent,omitempty"`
 
-	IntervalMillis     *int64 `yaml:"interval,omitempty" json:"interval,omitempty"`
+	// The interval for full backup in milliseconds.
+	IntervalMillis *int64 `yaml:"interval,omitempty" json:"interval,omitempty"`
+	// The interval for incremental backup in milliseconds.
```
Comment (korotkov-aerospike) [inline pkg/model/backup_routine.go:26]:
```suggestion
	// The interval for incremental backup in milliseconds (optional).
```

File: pkg/model/backup_routine.go [hunk 5]
```diff
@@ -5,24 +5,35 @@ import (
...
 	IncrIntervalMillis *int64 `yaml:"incr-interval,omitempty" json:"incr-interval,omitempty"`
 
-	Namespace string   `yaml:"namespace,omitempty" json:"namespace,omitempty"`
-	SetList   []string `yaml:"set-list,omitempty" json:"set-list,omitempty"`
-	BinList   []string `yaml:"bin-list,omitempty" json:"bin-list,omitempty"`
-	NodeList  []Node   `yaml:"node-list,omitempty" json:"node-list,omitempty"`
+	// The name of the namespace to back up.
+	Namespace string `yaml:"namespace,omitempty" json:"namespace,omitempty"`
+	// The list of backup set names.
```
Comment (korotkov-aerospike) [inline pkg/model/backup_routine.go:31]: // The list of backup set names (optional, an empty list implies backing up all sets.).

Comment (korotkov-aerospike) [inline pkg/model/backup_routine.go:31]:
```suggestion
	// The list of backup set names (optional, an empty list implies backing up all sets.).
```

File: pkg/model/backup_routine.go [hunk 6]
```diff
@@ -5,24 +5,35 @@ import (
...
+	SetList []string `yaml:"set-list,omitempty" json:"set-list,omitempty"`
+	// The list of backup bin names.
```
Comment (korotkov-aerospike) [inline pkg/model/backup_routine.go:33]:
```suggestion
	// The list of backup bin names (optional, an empty list implies backing up all bins).
```

File: pkg/model/backup_routine.go [hunk 7]
```diff
@@ -5,24 +5,35 @@ import (
...
+	BinList []string `yaml:"bin-list,omitempty" json:"bin-list,omitempty"`
+	// The list of nodes in the Aerospike cluster to run the backup for.
+	NodeList []Node `yaml:"node-list,omitempty" json:"node-list,omitempty"`
 
 	PartitionList *string `yaml:"partition-list,omitempty" json:"partition-list,omitempty"`
```
Comment (korotkov-aerospike) [inline pkg/model/backup_routine.go:38]: backup_status_init
```suggestion
        // List of partitions to back up. Partition filters can be ranges, individual partitions, or 
        // records after a specific digest within a single partition.
        // This argument is mutually exclusive to after-digest.
	PartitionList *string `yaml:"partition-list,omitempty" json:"partition-list,omitempty"`
```

File: pkg/model/backup_routine.go [hunk 8]
```diff
@@ -5,24 +5,35 @@ import (
...
 	AfterDigest   *string `yaml:"after-digest,omitempty" json:"after-digest,omitempty"`
```
Comment (korotkov-aerospike) [inline pkg/model/backup_routine.go:39]:
```suggestion
```

File: pkg/model/backup_policy.go (hunk 3 above)
Comment (reugn) [reply pkg/model/backup_policy.go:42]: The field is planned for removal.

File: pkg/model/restore_policy.go [hunk 9]
```diff
@@ -2,29 +2,67 @@ package model
...
+	DisableBatchWrites *bool `json:"disable-batch-writes,omitempty"`
+	// The max number of outstanding async record batch write calls at a time.
+	MaxAsyncBatches *uint32 `json:"max-async-batches,omitempty"`
+	// The max allowed number of records per an async batch write call.
+	// Default is 128 with batch writes enabled, or 16 without batch writes.
+	BatchSize *uint32 `json:"batch-size,omitempty"`
+	// Namespace details for the restore operation.
+	// By default, the data is restored to the namespace from which it was taken.
+	Namespace *RestoreNamespace `json:"namespace,omitempty"`
+	// The sets to restore.
```
Comment (korotkov-aerospike) [inline pkg/model/restore_policy.go:31]:
```suggestion
	// The sets to restore ( optional, an empty list implies restoring up all sets).
```

File: pkg/model/restore_policy.go [hunk 10]
```diff
@@ -2,29 +2,67 @@ package model
...
+	SetList []string `json:"set-list,omitempty"`
+	// The bins to restore.
```
Comment (korotkov-aerospike) [inline pkg/model/restore_policy.go:33]:
```suggestion
	// The bins to restore  (optional, an empty list implies restoring up all bins).
```

**Review by korotkov-aerospike — APPROVED** [review]

**Pull request merged**